src/components/BasicButton.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './BasicButton.scss';
+
+export default class BasicButton extends Component {

```

Comment:
```suggestion
export default class Button extends Component {

```
이 경우는 사이트 전체적으로 사용되는 버튼의 기본적인 형태기 때문에 이름을 `Button`으로 해주는 게 어떨까 싶은데요!

Reply:
네!